keyboards/pimoroni/keybow2040/config.h

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#define TRANSIENT_EEPROM_SIZE 1024
+
```

Comment:
This should not be needed once you switch from `transient` to the default EEPROM driver.
```suggestion
```

Reply:
Gone in round two